safe/impact_functions/volcanic/volcano_polygon_building/impact_function.py

```diff
@@ -46,6 +46,9 @@ def __init__(self):
         self.volcano_names = tr('Not specified in data')
         self._target_field = 'Hazard'
 
+        # From BuildingExposureReportMixin
+        self.building_report_threshold = 25
```

Comment:
Ah, yes. I read this http://python-history.blogspot.co.id/2010/06/method-resolution-order.html after reading your comment. Hmm, I am ok with the initialisation. :)